### ISSUE TITLE
Lagom detection failed to detect Lagom Scala correctly

### ISSUE THREAD

**Issue by fsat** [lightbend/sbt-reactive-app#47]
This causes the Java flavour of RP Service Discovery library is being added into dependency tree instead of the Scala one.

**Comment by fsat** [comment]
I think the current detection process based on the presence of plugin in the classpath may not be accurate.

The detection logic detects the presence of `LagomJava` plugin in the classpath of a Lagom scala project despite it not being enabled.

There are 2 alternatives I can think of which provides a better detection mechanism for Lagom at least.

**Detecting library dependencies**

For example, in Lagom Scala the dependency is as such:

```
[info]   |   +-com.lightbend.lagom:lagom-scaladsl-api_2.11:1.4.0-M3 [S]
[info]   |     +-com.lightbend.lagom:lagom-api_2.11:1.4.0-M3 [S]
[info]   |       +-com.google.guava:guava:22.0
[info]   |       | +-com.google.code.findbugs:jsr305:1.3.9
[info]   |       | +-com.google.errorprone:error_prone_annotations:2.0.18
```

Contrast to Lagom java:

```
[info]   | | | +-com.lightbend.lagom:lagom-javadsl-api_2.11:1.3.9 [S]
[info]   | | |   +-com.fasterxml:classmate:1.3.0
[info]   | | |   +-com.lightbend.lagom:lagom-api_2.11:1.3.9 [S]
[info]   | | |   | +-com.google.guava:guava:16.0.1 (evicted by: 19.0)
```

The downside of this would be the need to update this logic to match changes in Lagom artefact name (which I don't believe should change anymore now).

**Obtaining list of enabled plugins**

Another alternative is to obtain the list of `enabledPlugins` from the current project.

The name of the plugin classes used in the current detection can be matched with the list of `enabledPlugins` to ascertain which Lagom (or Play) version is enabled.

**Comment by longshorej** [comment]
Fixed via #48
